docs/modules/deployment/pages/core/centos-rhel/initialize-core.adoc

```diff
@@ -1,28 +1,41 @@
 .Configure PostgreSQL database access
```

Comment:
For some reason, this is not rendering correctly in my local build. Could just be a me thing.

Reply:
I think we have 2 options here:

1. Keep it as it is. We have the same issues in other places
2. Use a different way to highlight those "small titles"

But if we go with `2.` we should do this change also everywhere else:

a. step by step
b. in one step

I would vote for 2a :-) Since it is minimally invasive

But if we touch other parts later we should keep an eye on it. But this works only if we have a rule set we always follow I think. Otherwise we could miss it then. 

Are there other options?

Reply:
I like 2a as well. 
But for now, keep it as is, and let's create a separate Jira to do 2a, since it involves a lot of work going through all the docs.

Reply:
Done https://opennms.atlassian.net/browse/NMS-15603